handbook/engineering/code-insights/index.md

```diff
@@ -17,16 +17,43 @@ Code insights will also allow to combine these Sourcegraph-created metrics with
 
 Code insights is the first feature primarily built for non-search-based user personas.
 
-## Vision
-
-1. Expose the value of Sourcegraph's knowledge of your codebase to users at all levels of an organization.
-1. Expose generalizable metrics that let our users measure and track their own goals, whether those are migrations, code smells, test coverage,  security needs, cross-collaboration, or other information about code.
-1. Make these metrics scalable across thousands of repositories on a Sourcegraph instance.
-1. Expose features that let our users build their own insights.
-1. Expose features that let users drill down into insights by teams, repositories, or other sub-metrics.
-1. Connect code insights to other Sourcegraph features – make a campaign to take action on a metric or set up code monitoring to notify you about changes to an insight.
-1. Add plug-and-play integrations to Sourcegraph code insights that let users easily integrate third-party data into their insights.
-1. Discover new, validated code metrics that engineering teams should track (and let teams track them).
+## Goals 
+
+**Problem:** Code insights solves two problems. 
+
+1. The Sourcegraph sales cycle is unusual because although we consistently wow our users, the economic decision-maker is usually not one of these users. Instead, the people with the power to sign a contract with us are higher up within an organization and usually depend on running a trial to fully understand the value of Sourcegraph. If code insights answers these personas' higher-level questions about codebases that our core features do not currently answer, it could dramatically speed up our sales cycle as well as our sales pipeline. These personas . 
```

Comment:
There's. a trailing "These personas . " - not sure if that shouldn't be there or is an incomplete sentence.

Reply:
Ah, leftover from an edit – cleaned up, thank you!